src/sync/app.cpp

```diff
@@ -252,7 +252,6 @@ void App::UsernamePasswordProviderClient::reset_password(const std::string& pass
 
 void App::UsernamePasswordProviderClient::call_reset_password_function(const std::string& email,
                                                                        const std::string& password,
-                                                                       const std::string& args,
```

Comment:
This is mandatory for call_reset_password_function to work.

Reply:
It doesn't seem to be mandatory, given that the tests are exercising this successfully 😂 . Passing args in the request body results in a "invalid json" error message from the server. I can't seem to find any documentation on this endpoint, where can I find the source of truth?

Reply:
https://docs.mongodb.com/stitch/authentication/userpass/

Just send an array of strings (from the tests) instead of numbers, it will be simpler. The test passes successfully because leaving out the "arguments" field allows it to be considered an empty array.

Reply:
Fixed, thanks for the docs link!